main.js

```diff
@@ -0,0 +1,114 @@
+let BOOKS_DATA = [];
```

Comment:
- [ ] [OPTIONAL] I suggest we use always `const` with new Arrays and Objects. Plus, the capitalization convention you have used tells the next dev reading this code that BOOKS_DATA should be a constant. So, you can consider changing the `let` to `const`.